### PR TITLE
Prevent Blazor from being localized

### PR DESCRIPTION
**Pull request by Youssef1313**
Contributes to #22731

### CODE REVIEW

**Comment by guardrex** [comment]
FYI: Strings must be quoted.

File: docs/core/compatibility/aspnet-core/5.0/blazor-browser-support-updated.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ description: "Learn about the breaking change in ASP.NET Core 5.0 titled Blazor:
 author: scottaddie
 ms.author: scaddie
 ms.date: 10/01/2020
+no-loc: [Blazor, Blazor WebAssembly, Blazor Server]
```
Comment (Youssef1313) [inline docs/core/compatibility/aspnet-core/5.0/blazor-browser-support-updated.md:7]:
```suggestion
no-loc: [Blazor, "Blazor WebAssembly", "Blazor Server"]
```

**Comment by guardrex** [comment]
Thanks @Youssef1313 ... I'm going to ask Jason if we need to do this *everywhere* in the ASP.NET Core docs. I think the answer will be that we should. If so, I'm going to open an issue and address it everywhere at once (over there, not here on this doc set).

**Comment by gewarren** [comment]
> FYI: Strings (with spaces) must be quoted.

According to [this spec](https://review.docs.microsoft.com/en-us/new-hope/specs/markdown/ignore-localization?branch=master#implementations), the quotes aren't needed. But they don't hurt.

**Comment by guardrex** [comment]
It was recommended by @JasonCard, so we adopted it. I perhaps mistakenly thought that he was saying that it's a requirement when it actually isn't. My bad.

**Comment by guardrex** [comment]
Ok ... I see it. It was originally a misunderstanding of mine based on earlier remarks, but @JasonCard said that it seems happy either way ...

https://github.com/dotnet/AspNetCore.Docs/issues/19462#issuecomment-672104607

**Review by gewarren — APPROVED** [review]

**Pull request merged**